src/main/java/core/basesyntax/service/AmountFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface AmountFromFile {
+    List<Integer> getAmountInShop(List<String> fruits, List<FruitTransaction> fruitTransactions);
```

Comment:
The interface name 'AmountFromFile' is somewhat misleading. It suggests that the implementation will be dealing with amounts from a file, but the method signature indicates it deals with calculating amounts in a shop. Consider renaming the interface to better reflect its responsibility, such as 'ShopInventoryCalculator' or similar.